content/blog/2016/2016-06-16-parallel-test-executor-plugin.adoc

```diff
@@ -68,6 +68,14 @@ void runTests(def args) {
 void mvn(def args) { /* ... */ }
 ----
 
+[NOTE]
+====
+This pipeline expects to be run as part of a multibranch pipeline.
+To run it stand-alone, replace the `checkout scm` step with
```

Comment:
I think that it what Harpreet wanted - that it work without "Pipeline from SCM".   So, `git` step is correct.